src/backend/components/ProgressBar.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies.
+ */
+import classnames from 'classnames';
+
+const ProgressBar = ({
+	value,
+	max = 100,
+	className
+}) => {
+	const progress = Math.round( ( value / max ) * 100 );
```

Comment:
I see that some PHP code that can send `0` as a value for `max`.

I made some suggestions to make the backend send a valid number. But we can also put a guard here like

```js
const progress = 0 < max ? Math.round( ( value / max ) * 100 ) : 0;
```

Then we can remove things like `?.max || 100` as arguments for this Component usage.